zk-token-sdk/src/zk_token_proof_instruction.rs

```diff
@@ -306,6 +306,47 @@ pub enum ProofInstruction {
     ///   ii. `u32` byte offset if proof is provided as an account
     ///
     VerifyFeeSigma,
+
+    /// Verify a grouped-ciphertext with 3 handles validity proof.
+    ///
+    /// A grouped-ciphertext validity proof certifies that a grouped ElGamal ciphertext is
+    /// well-defined, i.e. the ciphertext can be decrypted by private keys associated with its
+    /// decryption handles.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   * Creating a proof context account
+    ///   0. `[writable]` The proof context account
+    ///   1. `[]` The proof context account owner
+    ///
+    ///   * Otherwise
+    ///   None
+    ///
+    /// Data expected by this instruction:
+    ///   `GroupedCiphertextValidityProofContext`
```

Comment:
This doesn't mention that it's possible to read from account data, like the other instructions. Can you add that in?

Reply:
Oh yes, I totally missed that! Thanks!